contracts/util/LogicVersionLib.sol

```diff
@@ -93,10 +124,13 @@ library LogicVersionLib {
         returns (address lastLogic, uint256 lastVersion)
     {
         requireExists(self);
-        lastLogic = self.logic;
-        lastVersion = self.version;
-        self.logic = address(0x0);
-        self.version = 0;
+        lastLogic = self.logicVersions[self.latestVersion];
+        lastVersion = self.latestVersion;
+
+        self.logicVersions[self.currentVersion] = address(0x0);
+        self.logicVersions[self.latestVersion] = address(0x0);
+        self.currentVersion = 0;
+        self.latestVersion = 0;
         self.exists = false;
     }
```

Comment:
I want to say that we should just remove the functionality to remove a logic version. I see no reason to have this functionality